c2corg_ui/views/document.py

```diff
@@ -366,9 +366,9 @@ def render_page(content1, content2):
                 'version1': version1,
                 'version2': version2,
                 'geometry1': doc_v1['geometry']['geom']
-                if doc_v1['geometry'] else None,
+                if 'geometry' in doc_v1 else None,
```

Comment:
I did not have any problem during my work. I am not sure if other documents contain geometry if they do not have any. For the articles, I think we did not put it to schema so the attribute should never be there.